src/main/java/com/labsynch/labseer/service/ParentAliasServiceImpl.java

```diff
@@ -42,13 +49,56 @@ public Parent updateParentAliases(Parent parent) {
 		return parent;
 	}
 
+	@Override
+	public void validateParentAliases(Set<ParentAlias> aliasesToBeSaved) throws NonUniqueAliasException {
+
+		// Check for unique parent aliases in what is being passed in
+		if (!propertiesUtilService.getAllowDuplicateParentAliases()) {
+			// Array of alias names in parent aliases
+			List<ParentAlias> duplicates = aliasesToBeSaved.stream()
+					.filter(alias -> !(alias.isIgnored() | alias.isDeleted()))
+					.collect(Collectors.groupingBy(ParentAlias::getAliasName))
+					.entrySet().stream()
+					.filter(e -> e.getValue().size() > 1)
+					.flatMap(e -> e.getValue().stream())
+					.collect(Collectors.toList());
+			if (duplicates.size() > 0) {
+				throw new NonUniqueAliasException("Duplicate parent aliases not allowed");
```

Comment:
nit: Can we include the alias strings that were duplicated in this error message?

Reply:
This was intentional as the summary would get messy if there were a lot of these messages. The summary html deduplicates the list and provides a message like "10 records had the error XXXX". This is the case with lots of other error messages during bulk loading and bulk loading validation.

Users would be able to determine the issue by looking at the summary files.